Src/Idioms.FsCheckUnitTest/Program.fs

```diff
@@ -0,0 +1 @@
+﻿module Program = let [<EntryPoint>] main _ = 0
```

Comment:
I'm curious to find out why we have to do this for a Class Library project.

Reply:
As far as I got it, test project for the .NET Core is not a library, rather it's an application. The key point is that the default template for the .NET Core test project includes this file.

You can find more detail here (that is the error I met):
https://github.com/Microsoft/visualfsharp/issues/2669
https://github.com/dotnet/netcorecli-fsc/issues/79